Sources/SymbolKit/SymbolGraph/Symbol/DeclarationFragments/DeclarationFragments+Simplify.swift

```diff
@@ -0,0 +1,167 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+fileprivate extension SymbolGraph.Symbol.DeclarationFragments.Fragment {
+    init(textFragment text: String) {
+        self.spelling = text
+        self.kind = .text
+        self.preciseIdentifier = nil
+    }
+}
+
+fileprivate extension UnifiedSymbolGraph.Symbol {
+    func declarationFragments(selector: UnifiedSymbolGraph.Selector) -> [SymbolGraph.Symbol.DeclarationFragments.Fragment]? {
+        return (self.mixins[selector]?[SymbolGraph.Symbol.DeclarationFragments.mixinKey] as? SymbolGraph.Symbol.DeclarationFragments)?.declarationFragments
+    }
+
+    func functionSignature(selector: UnifiedSymbolGraph.Selector) -> SymbolGraph.Symbol.FunctionSignature? {
+        return self.mixins[selector]?[SymbolGraph.Symbol.FunctionSignature.mixinKey] as? SymbolGraph.Symbol.FunctionSignature
+    }
+}
+
+internal extension SymbolGraph.Symbol {
+    func overloadSubheadingFragments(
+        argumentSeparator: String = ":",
+        printTrailingArgumentSeparator: Bool = true
```

Comment:
These parameters were meant to be available in case we expanded the implementation to other languages (like C++) in the future. I can take them out for now, though.